### PR TITLE
Add a blank space after all // in line comments

### PR DESCRIPTION
**Pull request by michpohl**
First: Thank you for building this plugin! Really like it!

Now to the PR content:
When using the plugin, I noticed that [ktlint](https://github.com/pinterest/ktlint) complained about the missing spaces in the line comments at the top of the generated `Secrets.kt` file (Adding a leading space is a very widely used pattern for comments, which is why ktlint flags this by default).

Screenshot 2022-05-17 at 09 20 16

So I decided this can easily be solved with a small PR. To maintain uniformity, I changed all line comments to this pattern:

Before: 
```kotlin
//This is my comment
```
After:
```kotlin
// This is my comment
```

This is a small change, but might save folks a little trouble that try to use this plugin in their CI/CD chain.

### CODE REVIEW

File: src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt [hunk 1]
```diff
@@ -99,10 +99,10 @@ open class HiddenSecretsPlugin : Plugin<Project> {
         @Input
         fun getKeyNameParam(): String {
             val chars = ('a'..'z') + ('A'..'Z')
-            // Default random key name
+            //  Default random key name
```
Comment (ben-j69) [inline src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt:102]: Double space

File: src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt [hunk 2]
```diff
@@ -194,7 +194,7 @@ open class HiddenSecretsPlugin : Plugin<Project> {
             object : Action<Copy?> {
                 @TaskAction
                 override fun execute(copy: Copy) {
-                    // in the case of buildSrc dir
+                    //  in the case of buildSrc dir
```
Comment (ben-j69) [inline src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt:197]: Double space

**Review by ben-j69 — CHANGES_REQUESTED** [review]
Hello and thanks for the PR.
I totally agree with these changes.

Can you just fix the 2 errors please ?

Thanks

File: src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt (hunk 1 above)
Comment (michpohl) [reply src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt:102]: > Hello and thanks for the PR. I totally agree with these changes.
> 
> Can you just fix the 2 errors please ?
> 
> Thanks

Oh, absolutely! My confidence in find & Replace was a little high apparently :-)

**Review by ben-j69 — APPROVED** [review]

**Pull request merged**